installation/single_machine_rhel.rst

```diff
@@ -31,7 +31,7 @@ Let's create directories for those nodes to store their data. For convenience in
   sudo su - postgres
 
   # include path to postgres binaries
-  export PATH=$PATH:/usr/pgsql-9.6/bin
+  export PATH=$PATH:/usr/pgsql-10.0/bin
```

Comment:
This should be ```/usr/pgsql-10/bin```

Reply:
K. That's how it was on the debian side too.